vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/framework/machines.go

```diff
@@ -63,22 +63,28 @@ func (f *Framework) DeleteMachineAndWait(machine *machinev1beta1.Machine, client
 	}
 }
 
-func (f *Framework) waitForMachineToRun(machine *machinev1beta1.Machine, client types.CloudProviderClient) {
+func (f *Framework) waitForMachineToRun(machine *machinev1beta1.Machine, client types.CloudProviderClient) *machinev1beta1.Machine {
```

Comment:
This method waits until a machine is running. It's not supposed to return anything. Rather, create another method retrieving a machine invoked after `waitForMachineToRun`.

Reply:
done.